src/state/actions/test.js

```diff
@@ -0,0 +1,74 @@
+
+import {
+  createAction,
+  requestLoginAction,
+  requestLoginActionSuccess,
+  requestLoginActionFailure,
+} from './index';
+
+import {
+  REQUEST_LOGIN_ACTION,
+  REQUEST_LOGIN_ACTION_SUCCESS,
+  REQUEST_LOGIN_ACTION_FAILURE,
+} from './actionTypes';
+
+describe('STATE - Actions', () => {
+  test('createAction', () => {
+    const mockedType = 'mockedType';
+    const mockedPayload = 'mockedPayload';
+    const mockedOptions = 'mockedOptions';
+
+    const expected = {
+      type: mockedType,
+      payload: mockedPayload,
+      options: mockedOptions,
+    };
+
+    expect(createAction(mockedType, mockedPayload, mockedOptions)).toEqual(expected);
+  });
+
+  test('requestLoginAction', () => {
+    const mockedEmail = 'mockedEmail';
+    const mockedPassword = 'mockedPassword';
+    const mockedRememberOption = true;
+
+    const action = requestLoginAction(mockedEmail, mockedPassword, mockedRememberOption);
+    const expectedAction = {
+      type: REQUEST_LOGIN_ACTION,
+      payload: {
+        email: mockedEmail,
+        password: mockedPassword,
+        rememberOption: true,
+      },
+      options: undefined,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://app.codacy.com/manual/iagoregiani/verificador-associacao/pullRequest?prid=5574829)